backend/config/config_loader.py

```diff
@@ -27,20 +31,46 @@ class ConfigDict(TypedDict):
     EXCLUDED_MULTI_PARTS_EXT: list[str]
     EXCLUDED_MULTI_PARTS_FILES: list[str]
     PLATFORMS_BINDING: dict[str, str]
+    ROMS_FOLDER_NAME: str
+    SAVES_FOLDER_NAME: str
+    STATES_FOLDER_NAME: str
+    SCREENSHOTS_FOLDER_NAME: str
+    HIGH_PRIO_STRUCTURE_PATH: str
+
+
+class Config:
+    EXCLUDED_PLATFORMS: list[str]
+    EXCLUDED_SINGLE_EXT: list[str]
+    EXCLUDED_SINGLE_FILES: list[str]
+    EXCLUDED_MULTI_FILES: list[str]
+    EXCLUDED_MULTI_PARTS_EXT: list[str]
+    EXCLUDED_MULTI_PARTS_FILES: list[str]
+    PLATFORMS_BINDING: dict[str, str]
+    ROMS_FOLDER_NAME: str
+    SAVES_FOLDER_NAME: str
+    STATES_FOLDER_NAME: str
+    SCREENSHOTS_FOLDER_NAME: str
+    HIGH_PRIO_STRUCTURE_PATH: str
```

Comment:
These need to be repeated twice, shame but necessary.

Reply:
why need to be repeated twice?

Reply:
there's no way to use the typedef in the class, or the other way around. even asked chatgpt and it had no clue lmao.